components/ui/Feed/FeedView.tsx

```diff
@@ -48,12 +51,18 @@ const SortIconType = {
   MostComments: <IconMessage />,
 };
 
+const ContextualMenuIconType = {
+  Ellipses: <IconDots />,
+  All: <IconWorld />,
+  Local: <IconMapPin />,
+  Subscribed: <IconBookmark />
+};
+
 function FeedView({ feed, community = false, header }: FeedViewProps) {
   // State Props
   // TODO Handle this
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [endReached, setEndReached] = useState(false);
-  const [, setSortIcon] = useState(SortIconType[feed.sort]);
```

Comment:
I believe the `useEffect` on line 83 is what actually causes the re-render, so I think deleting this state setter is no longer needed?

Reply:
Yea I think that was an old remnant. Good to get rid of it. (Lots of stuff to clean up as you can probably tell, that's the next item on the agenda really before feature implementation)